styles/_helpers.scss

```diff
@@ -55,6 +118,7 @@ Helpers - Co-op Front-end Toolkit
 .dark-grey-bg { background: $dark-grey; }
 .bluey-dark-grey { color: $bluey-dark-grey; }
 .bluey-dark-grey-bg { background: $bluey-dark-grey; }
+.added-to-page-bg { background: $added-to-page; }
 
 // Padding
```

Comment:
I know this would introduce a breaking change, but it may be a good idea to now split the layout helpers from the colour ones. They both have quite differently supported use cases, and loading all the colour classes every time seems a bit overkill. Thoughts?

Reply:
I don't think it's a massive problem. How often are people actually updating...? And we need to move forward regardless really.